### DISCUSSION TITLE
Only with a smartphone?

### DISCUSSION THREAD

**Discussion by villanyora** [eneshecan/whatsapp-for-linux#365] [Q&A] [answered]
For various reasons I'm not using a smartphone anymore, however I'd like to use whatsapp web. Is it possible, or only if one has a smartphone with whatsapp running on it?

**Comment by manuelselvaf** [comment, marked as answer]
Hello @villanyora, you need a "smartphone" every X days (around 3 weeks usually for me) just to associate your desktop or web version with a phone.

Like you I don't have a smartphone, and I set up a quite complex smartphone emulator on my computer for the association.
